test/integration/admin/api/member_permissions_controller_test.rb

```diff
@@ -0,0 +1,110 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class Admin::Api::MemberPermissionsControllerTest < ActionDispatch::IntegrationTest
+
+  setup do
+    @provider = Factory(:provider_account, self_domain: 'example.org')
+    FactoryGirl.create(:service, :id => 1, :account => @provider)
+    FactoryGirl.create(:service, :id => 2, :account => @provider)
```

Comment:
I won't ask you to `push` only for this, so if this is the only thing to change, feel free to just ignore it for now, but it would be nicer to have:
```ruby
@services = FactoryGirl.create_list(:service, 2, account: @provider) # I am not sure if "simple_service" instead of "service" would be enough as well :)
```
And later accessing those ids by `@services[0].id` and `@services[1].id`.
The reason is this provokes a random failure as it happened for another PR in Jenkins and was fixed by Josemi in https://github.com/3scale/porta/pull/359
The thing is if there is any other `service` with `id` 1 in the DB at this moment, this test will fail, so we should allow it to be auto-incremental 😄

Reply:
Aha, I've seen this also on one occasion!
It is weird though, as I was expecting the DB to cleanup before each test. I think I saw it somewhere in the code...🤔

Reply:
Yeah, it is supposed to clean between tests but sometimes it leaves the `master_account` there, we don't know why 🤷‍♀️ And also the previous line `Factory(:provider_account)` creates a service at some point I think and it could conflict 🤔 
Btw, I know there is old code and we copy and paste, but when you find the opportunity, it is better `FactoryGirl.create(...)` than `Factory(...)` 😄